src/content/docs/merge-queue/performance.mdx

```diff
@@ -141,19 +143,21 @@ Once you've input your parameters, the calculator will suggest the optimal
 configuration for your merge queue, ensuring an efficient and seamless merging
 process.
 
-<MergeQueueCalculator client:only="react"/>
+<MergeQueueCalculator client:only="react" />
+
 :::note
-  - This calculator optimizes CI usage and throughput but not latency. 
-    If you want to further optimize latency, you need to increase 
-    the number of speculative checks.
 
-  - The average latency computing assumes that the number of PR merged per hour 
-    enters the queue at the beginning of the hour.
+- This calculator optimizes CI usage and throughput but not latency.
+  If you want to further optimize latency, you need to increase
```

Comment:
same remark here with the spacing, idk if it's the linting or if it's voluntary ?